templates/referent/users_list.html.twig

```diff
@@ -188,7 +188,11 @@
                                 </small>
                                 <br />
                                 <small>
-                                    {{ user.isCommitteeSupervisor ? 'Animateur local' : user.isCommitteeHost ? 'Co-animateur local' : '' }}
+                                    {%- if user.isCommitteeSupervisor -%}
+                                        Animateur local
+                                        {{- user.isCommitteeHost ? ' / ' : '' -}}
```

Comment:
I would declare a var before the if: `{% set is_host = user.isCommitteeHost %}`.

Reply:
Done